core/src/main/java/bio/terra/pearl/core/service/kit/DataStudyManagerKitRequest.java

```diff
@@ -0,0 +1,34 @@
+package bio.terra.pearl.core.service.kit;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter @Builder
+public class DataStudyManagerKitRequest {
```

Comment:
since this class isn't persisted directly, could we just put a PepperKitAddress as a member and save all the field copying?

Reply:
I'm trying to serialize two different things: the sent-to address (saved in Juniper) and the web request to DSM. The sent-to address is a subset of the fields in the overall request and is not a separate structure, thus the copying. Alternatives:

* save the entire DSM request in Juniper instead of just the address
* use a bean copying utility library to copy the properties whose names match

Reply:
Devon, do you have any preference for how to resolve this?

Reply:
Now that I understand what you're doing, this seems fine.